docs/outputs.md

```diff
@@ -47,7 +47,7 @@ Below is an example of a minimal implementation in a basic standalone html page.
 The above assumes the following structure, which is the default provided in the Web Bundle generated from the web app:
 
 ```txt
-web
+www
```

Comment:
We need to mention the readme.txt, it has WordPress deployment information.

Reply:
Good point, thanks for catching this.

Reply:
It's mentioned higher up in the page, in the WordPress section.